src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -32,7 +32,7 @@ public Arm(ArmMap extendMap) {
     }
 
     public boolean intakeBelowGround() {
-        return pivotHeight < Math.cos(Math.toRadians(armAngle)) * (data.distanceInches + 42.3);
+        return pivotHeight - 5 < Math.cos(Math.toRadians(armAngle)) * (data.distanceInches + 42.3);
```

Comment:
The arm length should come from the map here.

I know we had trouble with this but this seems like the correct approach. We should test this more.